packaging/suse/rpm/trento-checks.spec

```diff
@@ -11,72 +11,77 @@
 # case the license is the MIT License). An "Open Source License" is a
 # license that conforms to the Open Source Definition (Version 1.9)
 # published by the Open Source Initiative.
-#
+
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
+
+%define         trento_dir %{_datadir}/trento
+%define         trento_checks_dir %{trento_dir}/checks
 Name:           trento-checks
+# Set by _service via OBS and GitHub action "CI"
 Version:        0
 Release:        0
 Summary:        Checks for the Trento checks engine
 License:        Apache-2.0
-URL:            https://github.com/trento-project/checks
-Source:         https://github.com/trento-project/checks/archive/refs/heads/main.tar.gz
 Group:          System/Monitoring
+URL:            https://github.com/trento-project/checks
+# Added by _service via OBS
+Source:         %{name}-%{version}.tar.gz
```

Comment:
Can you put Source0 here? I feel like it's a SUSE/openSUSE standard policy to actually number the sources even if it's just one